sky/packages/sky/lib/src/widgets/framework.dart

```diff
@@ -1098,6 +1102,13 @@ class StatefulComponentElement<T extends StatefulComponent, U extends State<T>>
     assert(_state._config == null);
     _state._config = widget;
     assert(_state._debugLifecycleState == _StateLifecycle.created);
+  }
+
+  U get state => _state;
+  U _state;
+
+  void firstBuild() {
```

Comment:
Will write a test that verifies that initState() is called at a time where you can use 'context' to walk up the tree.